### DISCUSSION TITLE
Ready to be finalized?

### DISCUSSION THREAD

**Discussion by jessnicwelch** [ornldaac/deltax_workshop_2024#27] [General]
@achri19 Does it look like this repo is final? I can update the README to have the correct links and remove the registration link. Anything else? I appreciate you making sure the file names look nice and consistent.

**Comment by jessnicwelch** [comment]
@achri19 Oh, and it looks like two slide decks got moved to the root... should those replace the ones in `/slides`?

**Comment by achri19** [comment]
I just uploaded Dongchen's slides so all slides should be there.

**Comment by jessnicwelch** [comment]
@achri19 I made a new branch ([jess_updates](https://github.com/ornldaac/deltax_workshop_2024/tree/jess_updates)) for my suggested changes to the names of the folders and files. You can check that these look correct.
